installers/conda/osx/create_bundle.sh

```diff
@@ -0,0 +1,271 @@
+#   $1 - The name of the mounted volume that, if it still exists, should be detached
+function cleanup() {
+  ensure_volume_detached "$1"
+  rm -f "$TEMP_OSA_SCPT"
+  rm -f "$TEMP_IMAGE"
+}
+
+# Attempt to detach the named volume. This will wait for a maximum of 5 minutes
+#   $1 - The name of the mounted volume that, if it still exists, should be detached
+function ensure_volume_detached() {
+  local volume_name="$1"
+  if [ ! -d /Volumes/"$volume_name" ]; then
+    return
+  fi
+
+  # It can quite often happen that the resource is busy on first try.
+  # We keep trying for several times
+  local counter=0
+  while [ $counter -lt $DETACH_MAX_TRIES ]; do
+    if hdiutil detach /Volumes/"$volume_name"; then
+      break
+    fi
+    (( counter+=1 ))
+    sleep $DETACH_WAIT_SECS
+  done
+}
+
+# Use otool/install_name_tool to change a dependent path
+# to a reexported library. The original is assumed to contain
+# an absolute path
+#   $1 - Full Path to the .so/.dylib to fix
+function fixup_reexport_dependent_path() {
+  local libpath=$1
+  local -r dependent_path=$(otool -l "${libpath}" | grep LC_REEXPORT_DYLIB -A 2 | grep name | awk '{print $2}')
+  local -r dependent_fname=$(basename "${dependent_path}")
+
+  echo "Fixing up LC_REEXPORT_DYLIB path in '$libpath'"
+  install_name_tool -change "${dependent_path}" @rpath/"${dependent_fname}" "${libpath}"
+}
+
+function trim_conda() {
+  local bundle_conda_prefix=$1
+  echo "Purging '$bundle_conda_prefix' of unnecessary items"
+  # Heavily cut down everything in bin
+  mv "$bundle_conda_prefix"/bin "$bundle_conda_prefix"/bin_tmp
+  mkdir "$bundle_conda_prefix"/bin
+  cp "$bundle_conda_prefix"/bin_tmp/Mantid.properties "$bundle_conda_prefix"/bin/
+  cp "$bundle_conda_prefix"/bin_tmp/mantid-scripts.pth "$bundle_conda_prefix"/bin/
+  cp "$bundle_conda_prefix"/bin_tmp/MantidWorkbench "$bundle_conda_prefix"/bin/
+  cp "$bundle_conda_prefix"/bin_tmp/python "$bundle_conda_prefix"/bin/
+  cp "$bundle_conda_prefix"/bin_tmp/pip "$bundle_conda_prefix"/bin/
+  sed -i "" '1s|.*|#!/usr/bin/env python|' "$bundle_conda_prefix"/bin/pip
+  # Heavily cut down share
+  mv "$bundle_conda_prefix"/share "$bundle_conda_prefix"/share_tmp
+  mkdir "$bundle_conda_prefix"/share
+  mv "$bundle_conda_prefix"/share_tmp/doc "$bundle_conda_prefix"/share/
+  # Removals
+  rm -rf "$bundle_conda_prefix"/bin_tmp \
+    "$bundle_conda_prefix"/include \
+    "$bundle_conda_prefix"/man \
+    "$bundle_conda_prefix"/mkspecs \
+    "$bundle_conda_prefix"/phrasebooks \
+    "$bundle_conda_prefix"/qml \
+    "$bundle_conda_prefix"/qsci \
+    "$bundle_conda_prefix"/share_tmp \
+    "$bundle_conda_prefix"/translations
+  find "$bundle_conda_prefix" -name 'qt.conf' -delete
+  find "$bundle_conda_prefix" -name '*.a' -delete
+  find "$bundle_conda_prefix" -name "*.pyc" -type f -delete
+  find "$bundle_conda_prefix" -path "*/__pycache__/*" -delete
+  find "$bundle_contents" -name '*.plist' -delete
```

Comment:
If these are unnecessary do we have long term plans to do this in the CMake layer for all platforms?

Reply:
I think conda packages up this stuff and as we've based this on just installing a Conda environment we'll need to do it here. It will also be slightly different for Windows but we can share code for Linux/Mac that should be very similar. At this stage nothing needs CMake anymore and these scripts can run just based on having the Conda packages available.
I suppose there is a question about whether these scripts should live in this codebase or not. Should we have a separate standalone-bundle repo? @thomashampson @Pasarus Thoughts?

Reply:
Other projects have these in a seperate repo such as Freecad (https://github.com/FreeCAD/FreeCAD-Bundle), I however would caution against, as it will allow us to re-use assets in this repo, such as other scripts (Conda ones) and assets (Such as the icons that are present in this repo).